platform/FEATURE_EXPERIMENTAL_API/FEATURE_PSA/TARGET_MBED_PSA_SRV/TESTS/crypto_init/test/CMakeLists.txt

```
@@ -0,0 +1,14 @@
+# Copyright (c) 2021 ARM Limited. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.19.0 FATAL_ERROR)
+
+set(MBED_PATH ${CMAKE_CURRENT_SOURCE_DIR}/../../../../../../../ CACHE INTERNAL "")
+set(TEST_TARGET mbed-platform-feature-psa-crypto-init)
+
+include(${MBED_PATH}/tools/cmake/mbed_greentea.cmake)
+
+project(${TEST_TARGET})
+
+mbed_greentea_add_test(
+    TEST_NAME ${TEST_TARGET})
```

```suggestion
mbed_greentea_add_test(TEST_NAME ${TEST_TARGET})
```
This is not very important, but the style in Mbed OS is either have the whole statement in a single line, or have the closing `)` in its own line if multiple lines are used.

---

Fixed for this and similar targets 👍